### ISSUE TITLE
BUG: Errors on import with PySide 6.8.0.1

### ISSUE THREAD

**Issue by larsoner** [pyqtgraph/pyqtgraph#3171]
This isn't your library's problem but wanted to make you aware in case users hit this issue:

https://bugreports.qt.io/projects/PYSIDE/issues/PYSIDE-2898?filter=allopenissues

```
../mne-qt-browser/mne_qt_browser/_pg_figure.py:46: in <module>
    from pyqtgraph import (
../virtualenvs/base/lib/python3.12/site-packages/pyqtgraph/__init__.py:208: in <module>
    from .graphicsItems.ColorBarItem import *
../virtualenvs/base/lib/python3.12/site-packages/pyqtgraph/graphicsItems/ColorBarItem.py:10: in <module>
    from .PlotItem import PlotItem
../virtualenvs/base/lib/python3.12/site-packages/pyqtgraph/graphicsItems/PlotItem/__init__.py:1: in <module>
    from .PlotItem import PlotItem
../virtualenvs/base/lib/python3.12/site-packages/pyqtgraph/graphicsItems/PlotItem/PlotItem.py:18: in <module>
    from ..LegendItem import LegendItem
../virtualenvs/base/lib/python3.12/site-packages/pyqtgraph/graphicsItems/LegendItem.py:11: in <module>
    from .PlotDataItem import PlotDataItem
../virtualenvs/base/lib/python3.12/site-packages/pyqtgraph/graphicsItems/PlotDataItem.py:13: in <module>
    from .ScatterPlotItem import ScatterPlotItem
../virtualenvs/base/lib/python3.12/site-packages/pyqtgraph/graphicsItems/ScatterPlotItem.py:68: in <module>
    Symbols['x'] = tr.map(Symbols['+'])
E   AttributeError: type object 'PySide6.QtGui.QPainterPath' has no attribute 'Element'. Did you mean: 'elementAt'?
```

Feel free to close, or leave open for visibility!

**Comment by Radiofreak1041** [comment]
I also encounter this problem with 6.8.0.1. Downgrading to 6.8.0 gives me a segmentation fault. 6.7.3 works so far.

**Comment by larsoner** [comment]
FYI there is an upstream fix in 6.8.1 and 6.8.0.2:

https://bugreports.qt.io/browse/PYSIDE-2898

So now I think we just need to wait for a release!
